src/com/dotcms/rest/api/v1/authentication/CreateJsonWebTokenResource.java

```diff
@@ -157,6 +159,22 @@ public final Response getApiToken(@Context final HttpServletRequest request,
         return res;
     } // authentication
 
+    protected int getExpirationDays(final int expirationDays) {
+
+
+        final int jsonWebTokenMaxAllowedExpirationDay =
+                Config.getIntProperty(LoginService.JSON_WEB_TOKEN_MAX_ALLOWED_EXPIRATION_DAYS, -1);
+
+        final int maxAllowedExpirationDays =
+                (jsonWebTokenMaxAllowedExpirationDay > 0 && (expirationDays > jsonWebTokenMaxAllowedExpirationDay))?
+                        jsonWebTokenMaxAllowedExpirationDay:
+                         expirationDays;
+
```

Comment:
shouldn't we write something in the file log if (expirationDays > jsonWebTokenMaxAllowedExpirationDay)?